resources/queries/targetedms/QCTraceMetric.sql

```diff
@@ -0,0 +1,2 @@
+SELECT 0 AS PrecursorChromInfoId, SampleFileId, metric.Name AS SeriesLabel, value as MetricValue, metric
```

Comment:
Please add a comment explaining how this query fits in. It's special and can't be used as-is. It needs the help of the Java code

Reply:
though I'm still not clear why we need this query file at all. Could we in-line this in the Java code that's building the query, so that instead of tacking on the "WHERE metric=x" it appends the whole thing? This approach seems like we have more pieces in different places, some of which aren't useful on their own

Reply:
I can definitely add a comment in this query. It is similar to other metric queries like TicArea  and we have to store the queryName in QCMetricConfiguration for the trace metrics and if we do in-line java approach, it would require a similar check to write a custom select (Select 0 as PrecursorChromInfoId) for trace metrics.

Reply:
Also, I am not clear on how other metric queries are different than this one in terms of using them.